src/pokenav_menu_handler_2.c

```diff
@@ -43,54 +43,54 @@ static u32 LoopedTask_ReturnToConditionMenu(s32 state);
 static bool32 AreMenuOptionSpritesMoving(void);
-static void sub_81CA448(struct Sprite ** sprites, bool32 a1);
+static void SetMenuOptionGfxInvisibility(struct Sprite ** sprites, bool32 a1);
 static void sub_81CA474(struct Sprite * sprite);
 static void sub_81CA4AC(struct Sprite * sprite);
 static void sub_81CA580(u8 taskId);
-static void sub_81CA640(void);
-static void sub_81CA6AC(struct Sprite * sprite);
-static void sub_81CA698(void);
+static void CreateMatchCallBlueLightSprite(void);
+static void SpriteCB_BlinkingBlueLight(struct Sprite * sprite);
+static void DestroyRematchBlueLightSpriteId(void);
 static void AddOptionDescriptionWindow(void);
 static void PrintCurrentOptionDescription(void);
 static void PrintNoRibbonWinners(void);
-static bool32 sub_81CA7C4(void);
-static void sub_81CA7D4(void);
-static void sub_81CA7F4(void);
-static void sub_81CA808(u8 taskId);
-static void sub_81CA818(void);
-static void sub_81CA850(void);
-static void sub_81CA864(void);
-static bool32 sub_81CA89C(void);
-static void sub_81CA8B0(u8 taskId);
-static void sub_81CA92C(void);
-static void sub_81CA994(void);
-static void sub_81CA9C8(void);
-static void sub_81CA9D8(void);
-static void sub_81CA9EC(u8 taskId);
-static void sub_81CAA3C(void);
-
-static const u16 gUnknown_0861FC78[] = INCBIN_U16("graphics/pokenav/bg.gbapal");
-static const u32 gUnknown_0861FC98[] = INCBIN_U32("graphics/pokenav/bg.4bpp.lz");
-static const u32 gUnknown_0861FCAC[] = INCBIN_U32("graphics/pokenav/bg.bin.lz");
-static const u16 gUnknown_0861FD4C[] = INCBIN_U16("graphics/pokenav/outline.gbapal");
-static const u32 gUnknown_0861FD6C[] = INCBIN_U32("graphics/pokenav/outline.4bpp.lz");
-static const u32 gUnknown_0861FFF4[] = INCBIN_U32("graphics/pokenav/outline_map.bin.lz");
+static bool32 IsDma3ManagerBusyWithBgCopy_(void);
+static void CreateMovingBgDotsTask(void);
+static void DestroyMovingDotsBgTask(void);
+static void Task_MoveBgDots(u8 taskId);
+static void CreateBgDotPurplePalTask(void);
+static void ChangeBgDotsColorToPurple(void);
+static void CreateBgDotLightBluePalTask(void);
+static bool32 IsTaskActive_UpdateBgDotsPalette(void);
+static void Task_UpdateBgDotsPalette(u8 taskId);
+static void SetupPokenavMenuScanlineEffects(void);
+static void DestroyMenuOptionGlowTask(void);
+static void ResetBldCnt(void);
+static void InitMenuOptionGlow(void);
+static void Task_CurrentMenuOptionGlow(u8 taskId);
+static void SetMenuOptionGlow(void);
+
+static const u16 sPokenavBgDotsPal[] = INCBIN_U16("graphics/pokenav/bg.gbapal");
+static const u32 sPokenavBgDotsTiles[] = INCBIN_U32("graphics/pokenav/bg.4bpp.lz");
+static const u32 sPokenavBgDotsTilemap[] = INCBIN_U32("graphics/pokenav/bg.bin.lz");
+static const u16 sPokenavDeviceBgPal[] = INCBIN_U16("graphics/pokenav/outline.gbapal");
+static const u32 sPokenavDeviceBgTiles[] = INCBIN_U32("graphics/pokenav/outline.4bpp.lz");
+static const u32 sPokenavDeviceBgTilemap[] = INCBIN_U32("graphics/pokenav/outline_map.bin.lz");
```

Comment:
Are the file names for these still accurate? Should be updated if not

Reply:
renamed
`graphics/pokenav/outline.*` to `graphics/pokenav/device_outline.*`
`graphics/pokenav/bg.*` to `graphics/pokenav/bg_dots.*`